setting-up-webauthn-mfa.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Setting up WebAuthn / Passkey multi-factor authentication
+url: /setting-up-webauthn-mfa
+previous: /setting-up-multifactor-authentication
+next: /setting-up-otp-mfa
+---
+
+<em class="t-gray">Setting up WebAuthn / Passkey multifactor authentication</em>
+
+Using WebAuthn, also called Passkey, for multifactor authentication (MFA) is
+the best way to protect your account from takeover. It's stronger and easier
+to use than [OTP codes](/setting-up-otp-mfa).
+
+## Prerequisite
+
+To use WebAuthn, you will need at least _one_ of the following:
+
+* A hardware security token (sometimes called a security key), such as a
+  YubiKey or Google Titan Key.
+* A browser that supports the "Passkey" standard. Up-to-date versions of
+  Chrome, Safari, Firefox and Edge all support this standard.
+
+"Passkey" is really just WebAuthn, but fully managed by the browser in
+cooperation with your operating system. Its underlying implementation works
+the same way as the hardware tokens do (ie, it's WebAuthn). The only
+difference is that a hardware token is a physical, removable device that
+plugs into your computer which is activated by touch. Passkey is instead
+activated by some operating system capability like facial recognition,
+fingerprint recognition or simply by providing your system password.
```

Comment:
Hmmm from my understanding, Passkeys are multi-device FIDO credentials. So whereas with standard webauthn, if you enabled using touch id on your mac ONLY that mac touch id would be the key, with passkeys all your apple devices become keys. It's not _quite_ the same as standard webauthn and I don't think it's fully rolled out.  However, wherever and for whomever it is available, it'll just "work" as the browser takes care of that as part of Webauthn. 

Another area for confusion is that, very often I see the term used as a shorthand for passwordless-sign-in (ex: https://fidoalliance.org/passkeys/). While we could expand to that in the future, we're not there yet. 

So all of this to say, I don't know if we need to mention passkeys at all? It's kind of mixing the message a bit and, since passkeys are built on top of webauthn, I feel like it's clear enough to _only_  use the term webauthn.

Reply:
I agree that it's confusing. My concern is that browsers are already positioning it all WebAuthn as "Passkey", especially Chrome. I'm not sure we can safely ignore the terminology.

Reply:
Should we leave the distinction in the titles, in that case?

Reply:
I still think having it in the title will lead users to think we have passwordless sign in (when I google "passkey", almost every result in the google preview describes them as a replacement for passwords), but if you if you'd prefer to keep it in the titles, I think that's a fine compromise!

Reply:
I think I'm convinced that we should remove references to Passkey for now, except perhaps in passing (something like "Passkey is usable as a security device"). If folks show up later asking where Passkey is we can update the docs then.